src/main/java/jarvis/model/Consult.java

```diff
@@ -44,4 +47,16 @@ public int hashCode() {
     public String toString() {
         return "Consult at " + getTimePeriod();
     }
+
+    @Override
+    public String getType() {
+        return "consult";
+    }
+
+    @Override
+    public JsonAdaptedConsult toJson() {
```

Comment:
This should be a constructor for `JsonAdaptedConsult` that takes in a `Consult` parameter. See `JsonAdaptedStudent` for similar implementation.

Same goes for other Lesson subclasses.